### PR TITLE
Fix/mobile view

### PR DESCRIPTION
**Pull request by shayan-golafshani**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring
# How Has This Been Tested?
- [x] open localHost
- [ ] test through Mocha/Chai
- [x] dev tools
# Checklist:
- [x] My code follows the Turing's guidelines of this project
- [x] I have performed a self-review of my own code
- [x] No console error messages

I fixed the mobile view via scss, and removed unnecessary comments

### CODE REVIEW

File: src/css/base.scss [hunk 1]
```diff
@@ -219,20 +215,51 @@ footer {
...
+  
   }
-  footer {
-  flex-direction: column;
+ 
+  #sidebar {
+    height: 100%;
+    overflow: none;
   }
+
```
Comment (saberg1) [inline src/css/base.scss:233]: Thats how you CSS like. GOD@!!@

File: dist/index.html [hunk 2]
```diff
@@ -13,8 +13,8 @@
     <nav>
       <img id="flag" src="./images/hungary.png" alt="Small image of the Hungarian flag">
       <div class="hotel-title" >
-        <p>The Grand Budapest Hotel</p>
-        <p> You won't leave Hungary, hungry for a better time</p>
+        <h1>The Grand Budapest Hotel</h1>
```
Comment (RMartin0717) [inline dist/index.html:16]: Nice job using some more appropriate semantic HTML

File: src/css/base.scss [hunk 3]
```diff
@@ -219,20 +215,51 @@ footer {
     background: white;
   }
 }
-@media (max-width:550px) {
+@media (max-width:700px) {
```
Comment (RMartin0717) [inline src/css/base.scss:218]: I'm seeing some issues with this break point and the login in getting a little funky as the screen gets smaller. let's take a look at this together

**Review by RMartin0717 — COMMENTED** [review]
Really love the dynamic units, but let's take a look at your break points and make sure everything still displaying on a variety of device sizes

**Pull request merged**